python/cudf/cudf/_lib/aggregation.pyx

```diff
@@ -1,10 +1,10 @@
 # Copyright (c) 2020-2024, NVIDIA CORPORATION.
 
 import pandas as pd
+import pylibcudf
```

Comment:
We should update python/cudf/pyproject.toml's tool.isort table to add pylibcudf to the `known_rapids` section.

Reply:
thx